src/everest/config/control_config.py

```diff
@@ -49,124 +50,171 @@ class ControlConfig(BaseModel):
         str,
         AfterValidator(no_dots_in_string),
         AfterValidator(not_in_reserved_word_list),
-    ] = Field(description="Control name")
+    ] = Field(
+        description=dedent(
+            """
+            The control group name.
+
+            Controls name must be unique.
+            """
+        )
+    )
     type: Literal["well_control", "generic_control"] = Field(
-        description="""
-Only two allowed control types are accepted
+        description=dedent(
+            r"""
+            Control group type.
 
-* **well_control**: Standard built-in Everest control type designed for field\
- optimization
+            Only two allowed control types are accepted:
 
-* **generic_control**: Enables the user to define controls types to be employed for\
- customized optimization jobs.
-"""
+            * `"well_control"`: Standard built-in Everest control type designed
+              for field optimization
+            * `"generic_control"`: Enables the user to define controls types to
+              be employed for customized optimization jobs.
+            """
+        )
     )
     variables: Annotated[
         ControlVariable,
         AfterValidator(_all_or_no_index),
         AfterValidator(unique_items),
-    ] = Field(description="List of control variables", min_length=1)
+    ] = Field(
+        description=dedent(
+            """
+            List of control variables.
+            """
+        ),
+        min_length=1,
+    )
     initial_guess: float | None = Field(
         default=None,
-        description="""
-Initial guess for the control group all control variables with initial_guess not
-defined will be assigned this value. Individual initial_guess values in the control
-variables will overwrite this value.
-""",
+        description=dedent(
+            """
+            Initial guess for the control group.
+
+            The initial guess value that is assigned to all control variables in
+            the control group unless overridden at the variable level.
```

Comment:
Other descriptions separate this slightly;
I think I prefer having this as a separate line
```
This can be overridden at the variable level.
```
```
This group default may be overridden by individual variables.
This group default may be overridden at the variable level.
```
You can consider wording.